user-service/src/test/resources/application.properties.example

```diff
@@ -1,18 +1,16 @@
 spring.application.name=user_service
 spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
 
-spring.datasource.url=jdbc:mysql://localhost:3306/bank_test
-
 spring.datasource.username=root
-spring.datasource.password=
+spring.datasource.password=root
 spring.jpa.database-platform=org.hibernate.dialect.MySQL8Dialect
 spring.jpa.generate-ddl=true
-spring.jpa.hibernate.ddl-auto=create
+spring.jpa.hibernate.ddl-auto=update
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.show-sql=true
 logging.level.org.hibernate.SQL=DEBUG
 logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
-server.port=8080
+server.port=8084
```

Comment:
zasto ovo?

Reply:
ovo je samo kada se pokrecu testovi, ostalo mi je od ranije (nije pravi port)

Reply:
skapirao sam, samo je random, da li ima neki odredjen razlog za to ili iz gasa

Reply:
iz gasa 🤫🧏🏻‍♂️